src/components/Event/EventBox.tsx

```diff
@@ -0,0 +1,82 @@
+import { Icon } from "@iconify/react";
+
+export interface EventBoxProps {
+  EventName: string;
+  Enroll: boolean;
+  CurrentEnrollment: number;
+  MaxEnrollment: number;
+  ThaiFaculty: string;
+  EngFaculty: string;
+  Date: string;
+  Time: string;
+  Location: string;
+}
+const EnrollBox: React.FC<{
+  CurrentEnrollment: number;
+  MaxEnrollment: number;
+}> = ({ CurrentEnrollment, MaxEnrollment }) => (
+  <div className="flex flex-row gap-2 text-white">
+    <div className="rounded-full bg-pink-500 px-2 py-1 ">
+      <p className="text-xs">ต้องลงทะเบียน/ Limited seats</p>
+    </div>
+    <div className="rounded-full border-2 border-white px-2 py-1">
+      <p className="text-xs">
+        {CurrentEnrollment} / {MaxEnrollment}
+      </p>
+    </div>
+  </div>
+);
+
+const EventBox = ({
+  EventName,
+  Enroll,
+  CurrentEnrollment,
+  MaxEnrollment,
+  ThaiFaculty,
+  EngFaculty,
+  Date,
+  Time,
+  Location,
+}: EventBoxProps) => {
```

Comment:
there are two ways of define a props type, one is to use `FC` generic as you've done at [R14-R17](https://github.com/isd-sgcu/oph66-frontend/pull/23/files#diff-38ae16de040485cdeeb1cd128e0e38036c457f4c89d37770ec12694ba9b82553R14-R17).

another one is here, no generic. 

please decide which way to go. my recommendation is using `FC` generic for concrete type. but it may cause a type issue for component which is then passed to a library component, such as `<Button component={} />` or `Formik` form for mismatching types. 

but never minds, I think it's not a case anymore.

feel free to choose regards to team's convention!